pyiron_base/job/util.py

```diff
@@ -122,7 +122,10 @@ def _get_project_for_copy(job, project, new_job_name):
 }
 
 
-def _get_safe_job_name(name, ndigits=8):
+def _get_safe_job_name(name, ndigits=8, special_symbols=None):
```

Comment:
```suggestion
def _get_safe_job_name(name: str, ndigits: Union[int, None] = 8, special_symbols: Union[Dict, None] = None):
```

You will probably also need ```from typing import Union, Dict```